lib/chef/provider/mount/linux.rb

```diff
@@ -57,6 +57,10 @@ def mounted?
             when /\A#{Regexp.escape(real_mount_point)}\s+#{device_mount_regex}\[/
               mounted = true
               logger.trace("Network device #{device_logstring} mounted as #{real_mount_point}")
+            # Permalink for network device mounted with a space in device name https://rubular.com/r/CK5zWWms96CRES
+            when /\A#{Regexp.escape(real_mount_point)}\s+#{device_sub_space_regex}\s/
+              mounted = true
+              logger.trace("Network device #{device_logstring} mounted as #{real_mount_point}")
```

Comment:
Lets add a comment here saying "see the comment above `device_with_space_escape` for an explanation of what's going on here" or some such.